runtime/brooklyn/src/pos/currency.rs

```diff
@@ -0,0 +1,965 @@
+// TODO: benchmark and set proper weight for calls
+
+use core::marker::PhantomData;
```

Comment:
It's a pallet. Let's extract it into common

Reply:
Fixed.